lib/cfnvpn/templates/lambdas/scheduler/slack.py

```diff
@@ -0,0 +1,49 @@
+import os
```

Comment:
@Guslington Could this and lib/cfnvpn/templates/lambdas/auto_route_populator/slack.py be a bit more dynamic and re-usable rather than duplicating the same boiler plate code?